### PR TITLE
Enable usability testing contributions

### PR DESCRIPTION
**Pull request by clarissalimab**
Creates a template issue and a contribution guide for usability tests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/usability_test.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Task 2
+
+#### What went well?
+Users intuitively clicked on the "?" icon to find the help documentation.
+
+#### What were the challenges?
+Some users didn't understand how to create a new blog post inside a folder, they first created the post and then edited the folder to include the post inside of it. They said they were looking for a select input to choose where the blog post should be placed instead of a folder icon.
+
+-->
```
Comment (clarissalimab) [inline .github/ISSUE_TEMPLATE/usability_test.md:65]: The placement for the results is just a suggestion. Some projects may prefer to have the results registered on the wiki or somewhere else. Personally, I like placing in the issues :smiley:

File: docs/contribution_guide_usability_testing.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### Test execution
+
+For each scenario, read it aloud to the user and watch they interact with the interface. Don't give hints, let the user try on their own. If they ask questions, guide them with other questions, be careful not to bias the test result by helping the user.
+
+After each scenario, you can ask some follow-up questions to gather some information that may come up only when the task is finished, like "where were you expecting to find that button first?", "which icon were you looking for?".
+
+## 3. Reporting results
+
+Add your results to the corresponding issue comments. Once all the tests are finished, add the conclusions to the issue template.
```
Comment (clarissalimab) [inline docs/contribution_guide_usability_testing.md:81]: Ditto about the placement of the results.

File: .github/ISSUE_TEMPLATE/usability_test.md (hunk 1 above)
Comment (wwahammy) [reply .github/ISSUE_TEMPLATE/usability_test.md:65]: I like issues as well!

File: docs/contribution_guide_usability_testing.md (hunk 2 above)
Comment (wwahammy) [reply docs/contribution_guide_usability_testing.md]: I like this!

**Pull request merged**